community/pike/APKBUILD

```diff
@@ -0,0 +1,235 @@
+}
+
+bzip2() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname bzip2 support"
+	depends="pike-core=$pkgver-r$pkgrel bzip2"
+	_install "usr/lib/pike/modules/*Bz2.*" "$subpkgdir"
+}
+
+dnssd() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname DNS SD support"
+	depends="pike-core=$pkgver-r$pkgrel"
+	_install "usr/lib/pike/modules/*DNS_SD.*" "$subpkgdir"
+}
+
+fuse() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname fUSE support"
+	depends="pike-core=$pkgver-r$pkgrel"
+	_install "usr/lib/pike/modules/*Fuse.so" "$subpkgdir"
+	_install "usr/lib/pike/modules/Fuse.pmod" "$subpkgdir"
+}
+
+gdbm() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname GDBM support"
+	depends="pike-core=$pkgver-r$pkgrel gdbm"
+	_install "usr/lib/pike/modules/*Gdbm.*" "$subpkgdir"
+}
+
+gl() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname OpenGL/GLUT support"
+	depends="pike-core=$pkgver-r$pkgrel freeglut mesa-gl libsm"
+	_install "usr/lib/pike/modules/*GL*" "$subpkgdir"
+}
+
+gtk() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname GTK2 support"
+	depends="pike-image=$pkgver-r$pkgrel libglade gtk+2.0"
+	_install "usr/lib/pike/modules/*GTK2*" "$subpkgdir"
+	_install "usr/lib/pike/modules/Tools.pmod/PV.pike" "$subpkgdir"
+}
+
+image() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname image handling modules"
+	depends="pike-core=$pkgver-r$pkgrel giflib-utils libjpeg tiff libwebp"
+	_install "usr/lib/pike/?.?/modules/Image.pmod" "$subpkgdir"
+	_install "usr/lib/pike/modules/Image.so" "$subpkgdir"
+	_install "usr/lib/pike/modules/*Image_[!S]*.so" "$subpkgdir"
+	_install "usr/lib/pike/modules/*Image*.pmod" "$subpkgdir"
+	_install "usr/lib/pike/modules/Graphics.pmod" "$subpkgdir"
+	_install "usr/lib/pike/modules/Colors.pmod" "$subpkgdir"
+	_install "usr/lib/pike/modules/Protocols.pmod/X.pmod/XImage.pmod" "$subpkgdir"
+}
+
+pcre() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname PCRE support"
+	depends="pike-core=$pkgver-r$pkgrel pcre"
+	_install "usr/lib/pike/modules/*Regexp_PCRE.*" "$subpkgdir"
+}
+
+sdl() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname SDL support"
+	depends="pike-image=$pkgver-r$pkgrel sdl sdl_mixer"
+	_install "usr/lib/pike/modules/*SDL.*" "$subpkgdir"
+}
+
+svg() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname SVG support"
+	depends="pike-core=$pkgver-r$pkgrel librsvg"
+	_install "usr/lib/pike/modules/_Image_SVG.so" "$subpkgdir"
+}
+
+sqlite() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname SQLite support"
+	depends="pike-core=$pkgver-r$pkgrel sqlite-libs"
+	_install "usr/lib/pike/modules/*SQLite.*" "$subpkgdir"
+	_install "usr/lib/pike/modules/Sql.pmod/sqlite*" "$subpkgdir"
+}
+
+doc() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname documentation"
+	depends="pike-core=$pkgver-r$pkgrel"
+	mkdir -p "$subpkgdir/usr/share/"
+	gzip usr/share/man/man1/pike.1
+	mv usr/doc/ "$subpkgdir/usr/share/"
+	mv usr/share/man/ "$subpkgdir/usr/share/"
+}
+
+dev() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname dev files"
+	depends="pike-core=$pkgver-r$pkgrel"
+	_install "usr/lib/pike/include/*.h" "$subpkgdir"
+	_install "usr/include/pike/*.h" "$subpkgdir"
+	mv "$subpkgdir/usr/lib/pike/include/regexp.h" "usr/lib/pike/include/"
+}
```

Comment:
You'd probably be better off letting the `default_dev` function do its thing.

Reply:
I actually need the `regexp.h` file inside `pike-core` package. Or will
```
dev() {
    default_dev
    mv .....
}
```
do the trick?

Reply:
Yep. That'll get it done.

https://github.com/alpinelinux/abuild/blob/c0a862930c1bd033be30f2c48e91a6f7e76a5a58/abuild.in#L1706

Reply:
If you want `pike-core` to split before `pike-dev` just change the order in the `subpackages` list.